recipes-nodes/lighthouse/init

```diff
@@ -65,8 +65,7 @@ start() {
 	chown $LIGHTHOUSE_USER:$ETH_GROUP $LOGFILE
 
 	# Ensure the LIGHTHOUSE_DIR exists and has correct permissions
-	mkdir -p $LIGHTHOUSE_DIR
-	chown -R $LIGHTHOUSE_USER:$ETH_GROUP $LIGHTHOUSE_DIR
+	install -d -m 0770 -o $LIGHTHOUSE_USER -g $ETH_GROUP $LIGHTHOUSE_DIR
```

Comment:
is there a reason why to do it like this and not explicit as the lines you removed?

Reply:
A single command that replaces the three commands from above (it was missing `chmod` as well). `install` is also explicit about the permissions and chmod

Reply:
I understand, I though the default permissions would be enough here and no need to give the extra permissions to the group beside the default ones.